carbon-p2-plugin/src/main/java/org/wso2/maven/p2/beans/CarbonArtifact.java

```diff
@@ -0,0 +1,113 @@
+/*
+ *  Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.wso2.maven.p2.beans;
+
+import org.apache.maven.artifact.Artifact;
+
+public class CarbonArtifact {
```

Comment:
Most classes in this pull request are obsolete. Assuming you are referring to class level javadocs, yes a class level javadoc is added to this class in a later PR.